runelite-client/src/main/java/net/runelite/client/Notifier.java

```diff
@@ -140,7 +142,7 @@ private Notifier(
 		// First check if we are running in launcher
 		this.terminalNotifierAvailable =
 			!Strings.isNullOrEmpty(RuneLiteProperties.getLauncherVersion())
-			&& isTerminalNotifierAvailable();
+				&& isTerminalNotifierAvailable();
```

Comment:
Revert this indentation change.
```suggestion
			&& isTerminalNotifierAvailable();
```